inputfiles/addedTypes.json

```diff
@@ -2207,6 +2207,150 @@
                         }
                     }
                 }
+            },
+            "MSFileSaver": {
```

Comment:
All changes here in addedTypes.json looks redundant. `implements` in `overridingTypes.json` should do the things automatically.

Reply:
All the added interfaces in addedTypes? such as MSFileSaver, MSNavigatorDoNotTrack, ExceptionInformation, etc.?

Reply:
Yup. They were once removed from the result because `implements` does not included them (so the code thought they were needless), but as you added them again via `overridingTypes.json`, now those should be kept intact without these manual typings.

Reply:
But, some interfaces like `MSLaunchUriCallback`, `ExceptionInformation`, `ConfirmSiteSpecificExceptionsInformation`, `StoreExceptionsInformation`, and `StoreSiteSpecificExceptionsInformation` have been deleted, maybe I need to add it back manually through `addedTypes.json`.

Reply:
That's because we have a type tracker that auto-deletes unreferenced/unexposed dictionaries and interfaces. Try removing them building again, and you'll see those types won't be removed again 😁